### DISCUSSION TITLE
CVE-2022-4565:When will the vulnerability be fixed?

### DISCUSSION THREAD

**Discussion by Tinsleyzh** [looly/hutool#2827] [Q&A]
When will the vulnerability be fixed? The Vulnerability Description mentions that this vulnerability has been fixed in **5.8.11**, but the this version is not found.
https://nvd.nist.gov/vuln/detail/CVE-2022-4565

**Comment by looly** [comment]
还没发布，最近发布。

**Reply by Tinsleyzh** [reply]
好的，大概啥时候呢？

**Reply by looly** [reply]
预计本周内，解决了剩余的issue。
